src/app/listener.ts

```diff
@@ -65,6 +65,11 @@ export class Listener {
       return;
     }
 
+    // If the member does not exist, it's probably the github bot
+    if (!this.container.guildService.get().members.cache.get(message.author.id)) {
+      return;
+    }
```

Comment:
There's a built in method for this.
```suggestion
    // If the member does not exist, it's probably the github bot
    if (message.author.bot) {
      return;
    }
```

Reply:
I purposely removed `message.author.bot` on an earlier commit for the visual tester

Reply:
Why? Aren't we checking for a bot?

Reply:
The tester is a bot, it needs to get a response

Reply:
So I think the issue is that the GitHub bot is from a webhook, and it doesn't have an associated user. I think a better way to do this would be to check if the message is from a webhook.